dashboard/app/views/devise/registrations/account_type.html.haml

```diff
@@ -1,3 +1,7 @@
+- content_for :head do
```

Comment:
@TurnerRiley and @kelbyhawn- this is how copilot helped me pull in the file so that it will load for the header instead of being delayed trying to load in the body and causing an error

Reply:
Does it still need to be loaded in the body then?

Reply:
If not, wondering if this line could also be in the partial 🤔

Reply:
Good call- moved the content_for into the partial and was able to just render at the top (and pull out the second call in the account_type and new_sign_up haml files).